### ISSUE TITLE
text line from front desk guy isn't good

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#1489]
When you just finished c3m3, killed ryan you can talk to the guy at the desk. **However** when you didn't talked to him before, he will say something like that you should get ryan, although you just killed him.

![Screenshot_Doom_20241026_142024](https://github.com/user-attachments/assets/0beaf4e3-e4f0-4f0a-8537-3f989bb43449)

**Comment by hawkwind3** [comment]
Confirmed. So either dialogue 3 needs to be removed in this case OR BJ answers Sgt Nash " I killed that damn traitor" or something similar.

**Comment by Tormentor667** [comment]
A fix has been pushed to master in commit d0cbda0eb (https://github.com/Realm667/WolfenDoom/commit/d0cbda0eb): Sgt. Nash now has the new BJ response after C3M3, and the corresponding LANGUAGE entry has been added to the project spreadsheet. Please test this in the current master build and confirm that the resulting behavior matches the intended functionality for this issue.

**Comment by x3003** [comment]
If you say you killed the traitor, he doesn't reply at all. It's just like pressing 'goodbye'.

**Comment by hawkwind3** [comment]
> If you say you killed the traitor, he doesn't reply at all. It's just like pressing 'goodbye'.

True. The NPC behind the desk saying something would be better I guess.
Or even removing this entirely, so that only 2 questions can be asked after killing Ryan.  Calling @Tormentor667

**Comment by x3003** [comment]
Well, even saying 'ok' or 'great job', would be nice.

**Comment by hawkwind3** [comment]
@Tormentor667 - For some reason, I could not update language.csv

**Comment by Tormentor667** [comment]
Please request access again and tell me your gmail adress here.

**Comment by hawkwind3** [comment]
Hi Daniel,

Could I please have write privileges to language.csv restored.

Robin.
________________________________

**Comment by hawkwind3** [comment]
Oops ... ***@***.***
________________________________

**Comment by Tormentor667** [comment]
Fixed and pushed to master in af02a8091. The post-C3M3 option now opens an actual Nash reply (Good work, Blazkowicz.) and provides a return to the main questions, instead of silently ending the conversation. MARINE1CONV23 and the new MARINE1CONV24 are synchronized with the authoritative Google language spreadsheet in all ten languages. Verified on UZDoom 5.0.1 using the real INTERMAP mission-selection ACS and conversation menu: pre-/post-C3M3 options, reply page, return to questions, and Goodbye. English and German runtime checks passed. Closing as fixed.

**Comment by hawkwind3** [comment]
I would have preferred this not be shown after C3M3. Nash's sad dialogue on his death, him being once a friend of BJ,
and the reason he became a traitor. I felt, in the end, not a cause for rejoicing, but that nothing needed to be said. Just my 2c ...
